packages/awsfirehose/data_stream/log/routing_rules.yml

```
@@ -0,0 +1,31 @@
+- source_dataset: awsfirehose.log
+  rules:
+    - target_dataset: aws.cloudtrail
+      if: ctx['aws.cloudwatch.log_stream'].contains('CloudTrail')
+      namespace:
+        - "{{labels.data_stream.namespace}}"
+        - default
```

Can users configure the namespace where logs are sent initially or are logs always sent to `logs-awsfirehose.log-default`?

If it's configurable, you may want to change this to

```suggestion
        - "{{data_stream.namespace}}"
```

Instead of hard-coding the namespace to `default`. Even if they can't configure it, changing to that suggestion should do no harm.

The same applies to the other routing rules.

---

Good question! Right now data are actually all going to `logs-generic-default` by default and we do have a config parameter user can use in firehose setting to change the data stream.